docs/dev/run_the_etl.rst

```diff
@@ -178,33 +178,33 @@ Add ``DAGSTER_HOME`` to the currecnt session with
 
     $ export DAGSTER_HOME=/path/to/dagster_home
 
-Once ``DAGSTER_HOME`` is set, launch Dagit by running:
+Once ``DAGSTER_HOME`` is set, launch the dagster UI by running:
 
 .. code-block:: console
 
-    $ dagit -m pudl.etl -m pudl.ferc_to_sqlite
+    $ dagster-webserver -m pudl.etl -m pudl.ferc_to_sqlite
```

Comment:
Is there a good reason to use `dagster-webserver` rather than `dagster dev` here? The latter launches the dagster daemon as well as the webserver right?

Reply:
That is correct. We currently don't have a use for the [dagster-daemon](https://docs.dagster.io/deployment/dagster-daemon):

> Several Dagster features, like [schedules](https://docs.dagster.io/concepts/partitions-schedules-sensors/schedules), [sensors](https://docs.dagster.io/concepts/partitions-schedules-sensors/sensors), and [run queueing](https://docs.dagster.io/guides/customizing-run-queue-priority), require a long-running dagster-daemon process to be included with your deployment.

I figured we should use `dagster-webserver` instead to limit the number of background processes running, given we aren't using the daemon features.